testing/acceptance_test.py

```diff
@@ -514,11 +515,20 @@ def test_pyargs_importerror(self, testdir, monkeypatch):
 
         result = testdir.runpytest("--pyargs", "tpkg.test_hello")
         assert result.ret != 0
-        # FIXME: It would be more natural to match NOT
-        # "ERROR*file*or*package*not*found*".
-        result.stdout.fnmatch_lines([
-            "*collected 0 items*"
-        ])
+
+        # Depending on whether the process running the test is the
```

Comment:
You can force `testdir` to call pytest as a subprocess by calling `testdir.runpytest_subprocess`... seems better than try to accommodate both possibilities for `testdir.runpytest`.